### ISSUE TITLE
Pagination is broken in web UI

### ISSUE THREAD

**Issue by kevinsbarnard** [fathomnet/community-feedback#142]
@ermbutler @hohonuuli 
Due to the Micronaut 3 -> 4 serialization changes, the API no longer has `totalPages` in pageable endpoint responses. This is causing several frontend issues with pagination.

The community page:
![image](https://github.com/fathomnet/community-feedback/assets/40082734/5ee87777-3c31-4825-83c8-50815208bcb4)

The submissions page:
![image](https://github.com/fathomnet/community-feedback/assets/40082734/f10abd05-3d1e-4ea9-8ba9-d4c21f6ad709)

The account management (admin) interface:
![image](https://github.com/fathomnet/community-feedback/assets/40082734/5a293bac-d872-45c4-861d-1673a58077dd)

**Comment by hohonuuli** [comment]
### Sanity check:

#### Request:

```http
GET http://fathomnet.org:8080/users?size=10&page=0
Authorization: Bearer <token>
```

#### Response

```http
HTTP/1.1 200 OK
date: Tue, 19 Sep 2023 21:26:19 GMT
Content-Type: application/json
content-encoding: gzip
content-length: 1832
connection: close

{
  "content": [
    // omitted for brevity
  ],
  "pageable": {
    "size": 10,
    "number": 0,
    "sort": {}
  },
  "totalSize": 500
}
```

**Comment by hohonuuli** [comment]
This is a micronaut serialization bug:

- [Bug Report](https://github.com/micronaut-projects/micronaut-serialization/issues/307)
- [Work around](https://github.com/micronaut-projects/micronaut-data/commit/1190f94e0286f773d21e805531f61cb23a7e18fa)

**Comment by hohonuuli** [comment]
From the API docs, `totalPages` is a default method on the [Page](https://micronaut-projects.github.io/micronaut-data/4.1.2/api/io/micronaut/data/model/Page.html) interface. It looks like it's not getting picked up by micronaut's serialzation anymore. (It previously was when we used straight Jackson). 

Also the work around proposed above is reported to no longer work.

It's probably best if we just roll our own totalPages method:

```javascript
function totalPages(pageSize, totalSize) {

  if (totalSize <= pageSize) {
    return 1;
  }

  return Math.ceil(totalSize / pageSize);
}
```

**Comment by hohonuuli** [comment]
Fixed in release 1.3.3. I've deployed the fix to the  fathomnet server.
